### PR TITLE
GDB-7279 See the product capabilities in wb

### PR DESCRIPTION
**Pull request by Emskiqq**
Add capabilities information to home page, license info page and upon registration page. 
Change graphDB version to 10.0.3-TR2 from 10.1.0-BOOTSTRAP-FTS-1.

### CODE REVIEW

**Comment by Emskiqq** [comment]
Jenkins retry build

**Comment by Emskiqq** [comment]
Jenkins retry build

**Review by desislava-hristova-ontotext — CHANGES_REQUESTED** [review]
Please, removed all changes caused by auto formatting of your IDE and turn off the auto formatting. This makes the MR hard to read. Leave only the relevant to the issue changes.

File: src/pages/home.html [hunk 1]
```diff
@@ -167,22 +186,38 @@ <h4 class="card-header">GraphDB {{getProductTypeHuman()}} {{'edition' | translat
...
+                        <div class="float-xs-left d-inline-block ml-1 mr-2 mb-1">
                             {{'maintenance.date' | translate}}
-                            <span class="data-value big">{{getLicense().latestPublicationDate ? (getLicense().latestPublicationDate | date : 'yyyy-MM-dd') : 'perpetual' | translate}}</span>
+                            <span
+                                class="data-value big">{{getLicense().latestPublicationDate ? (getLicense().latestPublicationDate | date: 'yyyy-MM-dd') : 'perpetual' | translate}}</span>
+                        </div>
+                        <div class="float-xs-left d-inline-block ml-1 mb-1">
+                            {{'capabilities' | translate}}
+                            <div class="data-value big">
+                                <div ng-if="getLicense().licenseCapabilities.length!==0">
```
Comment (desislava-hristova-ontotext) [inline src/pages/home.html:212]: Use space between the operator: getLicense().licenseCapabilities.length !== 0.

File: src/pages/home.html [hunk 2]
```diff
@@ -167,22 +186,38 @@ <h4 class="card-header">GraphDB {{getProductTypeHuman()}} {{'edition' | translat
...
+                                    <div ng-repeat="capability in getLicense().licenseCapabilities">
+                                        <div>{{capability}}</div>
+                                    </div>
+                                </div>
+                                <div ng-if="getLicense().licenseCapabilities.length===0">
```
Comment (desislava-hristova-ontotext) [inline src/pages/home.html:217]: Wrap the operator with spaces.

**Comment by Emskiqq** [comment]
Jenkins retry build

**Comment by desislava-hristova-ontotext** [comment]
Jenkins retry build

**Comment by Emskiqq** [comment]
Jenkins retry build

**Review by desislava-hristova-ontotext — APPROVED** [review]

**Pull request merged**